src/script/lua_api/l_minimap.cpp

```diff
@@ -117,6 +117,20 @@ int LuaMinimap::l_toggle_shape(lua_State *L)
 	return 1;
 }
 
+int LuaMinimap::l_set_shape(lua_State *L)
+{
+	LuaMinimap *ref = checkobject(L, 1);
+	Minimap *m = getobject(ref);
+	int shapenum = (int)lua_tonumber(L, 2);
+	if (!shapenum) {
+		return 0;
```

Comment:
Wouldn't this mean you can't set it to square? According to the documentation I could find `lua_tonumber` returns 0 if it can't convert the number.